.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
Will this enable dependabot on the repo? (not a huge fan of having it running, also because it will start to open pull requests in 18.300 forks).

Reply:
yeah if 18k forks rebase this would happen :man_shrugging:

Reply:
ref https://github.com/dependabot/dependabot-core/issues/2804 for the tracking of this issue

Reply:
We could use renovate instead: https://docs.renovatebot.com/modules/manager/github-actions/